### DISCUSSION TITLE
CVE-2024-33663 with python-jose requirement vulnerability

### DISCUSSION THREAD

**Discussion by dmvieira** [open-webui/open-webui#7581] [General]
### Summary
This is a [critical severity security](https://vuldb.com/?id.262059) issue [opened here](https://github.com/mpdavis/python-jose/issues/346). 

### Details
Latest version of openwebui is using python-jose 3.3.0

### PoC
look at requirements.txt or pyproject.toml

### Impact
Security scans are blocking it

**Comment by tjbck** [comment]
I'm pretty sure it's not being used anywhere in the codebase. Confirmation wanted here!

**Comment by dmvieira** [comment]
it's here: https://github.com/search?q=repo%3Aopen-webui%2Fopen-webui%20python-jose&type=code

**Comment by tjbck** [comment]
Yes it's in the requirements, but I believe it's not being used. Confirmation wanted here.

**Comment by nicksterling** [comment]
If it's not being used would it be safe to remove as a pip dependency?
